core/services/ocr2/plugins/ocr2keeper/integration_test.go

```diff
@@ -178,7 +179,7 @@ func getUpkeepIdFromTx(t *testing.T, registry *keeper_registry_wrapper2_0.Keeper
 	return parsedLog.Id
 }
 
-func TestIntegration_KeeperPlugin(t *testing.T) {
+func TestIntegration_KeeperPluginBasic(t *testing.T) {
```

Comment:
is there some annotation we can add here so that these tests only run for race detection?